docs/examples/hpo_launcher.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -v
+
+# Usage:
+#
+#   sbatch --array=0-100 --gres=gpu:1 --cpus-per-gpu=2 --mem-per-gpu=16G scripts/hpo_launcher.sh train.py
+#
+
+# Slurm configuration
+# ===================
+
+#SBATCH --exclude=kepler4,kepler3
```

Comment:
Is it possible to remove this very specific exclude?
Would it make more sense to specify `--gres=gpu:rtx8000:1` as an example instead?

Reply:
`kepler[3,4]` are still in production. They should be removed this year however.